modules/ui/src/main/scala/lucuma/ui/enums/Theme.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.enums
+
+import cats.effect.Sync
+import cats.syntax.all.given
+import lucuma.core.util.Enumerated
+import org.scalajs.dom
+import react.common.style.Css
+
+enum Theme(val tag: String, val clazz: Css):
+  case Light extends Theme("light", Css("light-theme"))
+  case Dark  extends Theme("dark", Css("dark-theme"))
+
+  def setup[F[_]](using F: Sync[F]): F[Unit] =
+    F.delay {
+      dom.document.body.classList.add(this.clazz.htmlClass)
+      Theme.values
+        .filterNot(_ === this)
+        .foreach(otherTheme => dom.document.body.classList.remove(otherTheme.clazz.htmlClass))
+    }
+
+object Theme:
+  val Default: Theme = Theme.Light
```

Comment:
Default to light theme?

Reply:
Ah, that was for observe. Which I guess can have a default dark now. Changing it.